src/systems/filecoin_blockchain/storage_power_consensus/_index.md

```diff
@@ -44,54 +46,124 @@ More precisely,
 - Nominal power = claimed power, unless the miner is in DetectedFault or Challenged state. Nominal power is used to determine total network storage power for purposes of consensus minimum.
 - Consensus power = nominal power, unless the miner fails to meet consensus minimum, or is undercollateralized.
 
-{{<label tickets>}}
-## Tickets
-
-Tickets are used across the Filecoin protocol as sources of randomness:
+{{<label beacon_entries>}}
+## Beacon Entries
+
+The Filecoin protocol uses randomness produced by a {{<sref drand>}} beacon to seed unbiasable randomness seeds for use in the chain (see {{<sref randomness>}}).
+
+In turn these random seeds are used by:
+
+- The {{<sref sector_sealer>}} as SealSeeds to bind sector commitments to a given subchain.
+- The {{<sref post_generator>}} as PoStChallenges to prove sectors remain committed as of a given block.
+- The Storage Power subsystem as randomness in {{<sref leader_election>}} to determine their eligibility to mine a block.
+
+This randomness may be drawn from various Filecoin chain epochs by the respective protocols that use them according to their security requirements.
+
+It is important to note that a given Filecoin network and a given drand network need not have the same round time, i.e. blocks may be generated faster or slower by Filecoin than randomness is generated by drand.
+For instance, if the drand beacon is producing randomness twice as fast as Filecoin produces blocks, we might expect two random values to be produced in a Filecoin epoch, conversely if the Filecoin network is twice as fast as drand, we might expect a random value every other Filecoin epoch. Accordingly, depending on both networks' configurations, certain Filecoin blocks could contain multiple or no drand entries.
+
+In all cases, Filecoin blocks must include all drand beacon outputs generated since the last epoch in the `BeaconEntries` field of the block header. Any use of randomness from a given Filecoin epoch should use the last valid drand entry included in a Filecoin block, as follows:
+
+```go
+GetRandomnessForEpoch(epoch) []BeaconEntry {
+    var rand Randomness
+    for rand == nil {
+        seeds := GetRandomSeedsForEpoch(epoch)
+        if seeds == nil {
+            epoch -= 1
+        } else {
+            rand = seeds[len(seeds)-1].Randomness
+        }
+    }
+    return rand
+}
```

Comment:
Although this may be what it's currently implemented, we already mentioned in different places that it's just a temporary resolution, since this leads to multiple randomness for the same height in absence of blocks.

We can decide to leave it like this and expect a PR that proposes a fix.

Reply:
Make it a FIP. If we already have a solution that's great and I can update. If not, the spec should represent the state of the protocol... but it's worth noting indeed. (made a note of it).

Reply:
yeah, this solution won't solve it (a block could return multiple randomnesses based on current and previous consecutive null blocks), I agree it will be an FIP.